libs/service-portal/health/src/screens/Payments/Payments.graphql

```diff
@@ -0,0 +1,91 @@
+query getCopaymentStatus {
```

Comment:
Might be wise to do another design pass over the grahpql domain if you need like a million queries to get the data